File: baekjoon/java/02525. 오븐 시계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int h = sc.nextInt();
        int m = sc.nextInt();
        int dur = sc.nextInt();
        m = m+dur;
        while (m>=60) {
            if (h == 23) h = 0;
            else h++;
            m = m - 60;
        }
        System.out.println(h + " " + m);
        sc.close();
    }

}

File: baekjoon/cpp/02525. 오븐 시계.cpp
#include <iostream>
using namespace std;

int main(void){
    int h, m, dur;
    cin >> h >> m;  // h : 0~23, m : 0~59
    cin >> dur;
    m = m+dur;
    while (m>=60) {
        if (h == 23) h = 0;
        else h++;
        m = m - 60;
    }

    cout << h << " " << m;
}
